packages/react/carbon.yml

```diff
@@ -0,0 +1,699 @@
+      externalDocsUrl: https://www.carbondesignsystem.com/components/text-input/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-textinput
+    theme:
+      status: stable
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-theme--default
+    tile:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/tile/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-tile
+    time-picker:
+      status: stable
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-timepicker
+    toggle:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/toggle/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-toggle
+    toggletip:
+      name: Toggletip
+      framework: react
+      externalDocsUrl: https://www.carbondesignsystem.com/components/toggletip/usage/
+      status: stable
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-toggletip
+      # TODO: @Aubrey - double check tags
+      tags:
+        - content-element 
+    tooltip:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/tooltip/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-tooltip
+    tooltip-definition:
+      status: stable
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-definitiontooltip
+    tooltip-icon:
+      name: Tooltip icon
+      framework: react
+      status: stable
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-tooltipicon
+      # TODO: @Aubrey - double check tags
+      tags:
+        - content-block
```

Comment:
Without any description this one is hard to categorize. I suggest tagging it with media and data-display. Delete content-block. Add input control IF this is ever used as a button.